### PR TITLE
Add 'bundle' directory and make Hera not download CRTM coeffs

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
What the title says. Ecbuild should now link from sorc/ to bundle/ and not complain about unable to link a directory

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -182,4 +182,28 @@ install*/
...
+/bundle/jedicmake
+/bundle/land-imsproc
+/bundle/land-jediincr
+/bundle/mom6
+/bundle/oops
+/bundle/saber
+/bundle/soca
+/bundle/test-data-release/
+/bundle/ufo
+/bundle/vader
```
Comment (aerorahul) [inline .gitignore]: Is `bundle` not sufficient?

**Review by aerorahul — DISMISSED** [review]
Looks good. 
Thanks for taking care of both.

File: .gitignore (hunk 1 above)
Comment (CoryMartin-NOAA) [reply .gitignore]: we want `/bundle/CMakeLists.txt` :-)

Comment (aerorahul) [reply .gitignore]: :-)

One can add `bundle/.gitignore` and put
```
*
!CMakeLists.txt
```
in it.

or in the top-level `.gitignore`

```
bundle/*
!bundle/CMakeLists.txt
```

Comment (CoryMartin-NOAA) [reply .gitignore]: I didn't know that ! trick, that's nice! Thanks!

**Review by aerorahul — APPROVED** [review]
looks good.

**Review by guillaumevernieres — APPROVED** [review]

**Review by DavidNew-NOAA — APPROVED** [review]

**Pull request merged**